docs/docs/en/guide/task/kubernetes.md

```diff
@@ -16,17 +16,17 @@ K8S task type used to execute a batch task. In this task, the worker submits the
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Namespace        | The namespace for running k8s task.                                                                                                                                                                                                                                                            |
+| Min CPU          | Minimum CPU requirement for running k8s task.                                                                                                                                                                                                                                                  |
+| Min Memory       | Minimum memory requirement for running k8s task.                                                                                                                                                                                                                                               |
+| Image            | The registry url for image.                                                                                                                                                                                                                                                                    |
+| Command          | The container execution command (yaml-style array), for example: ["printenv"]                                                                                                                                                                                                                  |
+| Args             | The args of execution command (yaml-style array), for example: ["HOSTNAME", "KUBERNETES_PORT"]                                                                                                                                                                                                 |
+| Custom label     | The customized labels for k8s Job.                                                                                                                                                                                                                                                             |
+| Node selector    | The label selectors for running k8s pod. Different value in value set should be seperated by comma, for example: `value1,value2`. You can refer to https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/node-selector-requirement/ for configuration of different operators. |
```

Comment:
Do we need to update the Chinese version doc?

Reply:
It's a typo error and no need to modify the Chinese doc.